lib/views/AreaGraph.js

```diff
@@ -0,0 +1,56 @@
+import Graph from '../views/Graph';
```

Comment:
This is the other big change - supporting an area graph (basically an envelope of points)